action/messageHubProduce.py

```diff
@@ -83,23 +83,20 @@ def main(params):
     # we successfully connected and found the topic metadata... let's send!
     if producer is not None:
         try:
-            logging.info("Producing message")
+            value = validatedParams['value']
```

Comment:
Could maybe require a `values` parameter instead of `value` for batched messages. Would need to error if arguments are passed for both parameters though.